### ISSUE TITLE
Add method to return compressed data

### ISSUE THREAD

**Issue by mhr3** [mjibson/esc#46]
Would be great if there was `FSGzippedByte` method which would return the compressed representation of the data, afterall pretty much every browser sends `Accept-Encoding: gzip`.

**Comment by mjibson** [comment]
Yes, I agree. I'm unlikely to add this myself in the near future though.

**Comment by andig** [comment]
@mhr3 any idea how to use such a method without implemented the http server side, too?

**Comment by mhr3** [comment]
I'm afraid you'd have to implement some http glue as well, don't think it's possible to make it work automatically.
